core/garment/src/garment.ts

```diff
@@ -1037,8 +1037,6 @@ async function garmentFromWorkspace(
         ...restRunners[runner.name]
       };
 
-      validateOptions(runner, options);
```

Comment:
@beshanoe The `validateOptions` invocation in the pic below seems to correctly validate both batch and non-batch handlers.  Can you see any reason for having validation in `runActionsInBatch` as well?  The options it is passing here do not have the options for each project, since those options are inside the batch variable.  That's why it's failing.

![image](https://user-images.githubusercontent.com/34415120/106313279-d7fc8d80-625f-11eb-8d13-0eb3d855d5f3.png)

Reply:
yeah, seems your right, all of those options are already merged in the `getActionGraph` and then validated in this place, so another place seems redundant(and also erroneous)